### DISCUSSION TITLE
Source of the inference model

### DISCUSSION THREAD

**Discussion by fherb2** [mcguirepr89/BirdNET-Pi#804] [General]
Hi,

I saw this project is forked from https://github.com/kahst/BirdNET-Lite from Stefan Kahl. But I found some deprecation hints.

My question: Who is the responsible to generate the inference model? Is it possible to get the sources of the model structures and the labeled source data? Is the used model part of the BirdNetPi project?

Normally the source data like the sounds here are the input of the AI model. Some pre processing like FFT is before. Who (or which project repository) can me get the information about the details of the AI modelling used by BirdNetPi? 

Best regards, Frank

**Comment by ehpersonal38** [comment]
I recommend contacting Stefan Kahl, who is the maintainer/creator of the latest BirdNET model project. https://github.com/kahst/BirdNET-Analyzer
